bin/selenium-standalone

```diff
@@ -53,7 +53,11 @@ function parseCommandAndOptions(javaPath) {
   // Merge default options, options from config file then command line options
   options = merge({}, defaultConfig, configFromFile, options);
 
-  options.seleniumArgs = seleniumArgs;
+  if (seleniumArgs.length) {
```

Comment:
I can do 
```js
options.seleniumArgs = seleniumArgs.length ? seleniumArgs : (Array.isArray(options.seleniumArgs) ? options.seleniumArgs : []);
```

But I'm not sure it would help readability.